File: Ruby/api_GET_transactions.rb
require 'uri'
require 'net/http'

url = URI("https://merchant.atomicpay.io/api/v1/transactions?crypto_code=REPLACE_WITH_VALUE&dateStart=REPLACE_WITH_VALUE&dateEnd=REPLACE_WITH_VALUE")

http = Net::HTTP.new(url.host, url.port)

request = Net::HTTP::Get.new(url)
request["Content-Type"] = 'application/json'
request["Authorization"] = 'Basic REPLACE_WITH_BASE64_ENCODED_CREDENTIALS'
request["cache-control"] = 'no-cache'

response = http.request(request)
puts response.read_body

File: NodeJS/api_GET_transactions.js
var request = require("request");

var options = { method: 'GET',
  url: 'https://merchant.atomicpay.io/api/v1/transactions',
  qs:
   { crypto_code: 'REPLACE_WITH_VALUE',
     dateStart: 'REPLACE_WITH_VALUE',
     dateEnd: 'REPLACE_WITH_VALUE' },
  headers:
   { 'cache-control': 'no-cache',
     Authorization: 'Basic REPLACE_WITH_BASE64_ENCODED_CREDENTIALS',
     'Content-Type': 'application/json' } };

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});
